README.md

```diff
@@ -140,7 +150,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+The schema define's the tables and columns in a database.
 ```
```

Comment:
What about the second part of the question? How do we represent a one-to-many relationship in a schema?